src/ncp/rcp_host.hpp

```diff
@@ -194,9 +188,19 @@ class RcpHost : public MainloopProcessor, public ThreadController
 
     ~RcpHost(void) override;
 
-    // Thread Control APIs
+    // Thread Control virtual methods
     void GetDeviceRole(const DeviceRoleHandler aHandler) override;
```

Comment:
Maybe in a future PR: make this parameter either a non-const value or a const reference.

Reply:
SG. Let's do it in a seperate PR.